### DISCUSSION TITLE
Atom Lite Went Rogue

### DISCUSSION THREAD

**Discussion by gcortes** [ESPresense/ESPresense#1233] [Q&A]
I have five Atom Lites running esprense. I noticed that Home Assistant was reporting I was always in one room regardless of how close I was to any of the other Atom Lites. I disconnected the one that always was reported as my location. The other ones then reported normally. I don't think this is an espresence issue. I can log into the problem Atom lite and have verified that the range is set correctly. I have a bunch of surplus Atom Lites so I'll probably replace it. I wanted to check here to see if any one else has had this problem. It had been working fine for at least two years.

**Comment by raptile** [comment]
you can lower the rssi which will improve a lot. don't forget to do some fine tuning of the sensors. calibrating the best u can do will help a lot.
